code/android-core-library/src/phone/java/com/adobe/marketing/mobile/AndroidLoggingService.java

```diff
@@ -0,0 +1,32 @@
+package com.adobe.marketing.mobile;
```

Comment:
nit: We can remove this class. I don't think any extension outside Core is using this.